diff --git a/lua/key-analyzer/main.lua b/lua/key-analyzer/main.lua
index 176fb11..e528013 100644
--- a/lua/key-analyzer/main.lua
+++ b/lua/key-analyzer/main.lua
@@ -136,12 +136,6 @@ local function create_keyboard_visual(maps, mode, modifier)
     table.insert(lines, string.format("Key: %s", modifier == "leader" and "<leader>" or modifier))
     table.insert(lines, "") -- Empty line
 
-    -- Shameless plug :(
-    table.insert(lines, "For more vim: https://x.com/OtivDev")
-    table.insert(highlights, { group = config_highlights.promo_highlight, pos = { 9, 0, 50 } })
-
-    table.insert(lines, "") -- Empty line
-
     return lines, highlights
 end
 
